lib/packages/docutils/rstgen.nim

```diff
@@ -195,26 +197,26 @@ proc addRtfChar(dest: var string, c: char) =
   else: add(dest, c)
 
 proc addTexChar(dest: var string, c: char) =
+  # Escapes 10 special Latex characters. Note that [, ], and ` are not
+  # considered as such. TODO: neither is @, am I wrong?
   case c
   of '_': add(dest, "\\_")
```

Comment:
factor everything that can be factored via:
```nim
of '_', '{', ...: add(dest, "\\" & c)
```

Reply:
ok, done.